models/factors/model.npz.decoder.yml

```diff
@@ -0,0 +1,12 @@
+relative-paths: true
+models:
+  - model.npz
+vocabs:
+  - vocab.en.fsv
+  - vocab.de.fsv
+beam-size: 12
```

Comment:
I haven't run tests yet, but can we use a smaller beam size to run them faster, or is beam size of 12 required for some reasons?

Reply:
No 12 is not required, I'll reduce the value